exercises/the-dice/canonical-data.json

```diff
@@ -0,0 +1,41 @@
+{
+  "exercise": "the-dice",
+  "version": "1.0.0",
+  "cases": [
+    {
+      "description": "two pairs of 6s",
+      "property": "calculatePairs",
+      "trial": [1, 1, 6, 6, 2, 6, 6],
```

Comment:
this is implicitly also testing the ability to find 6s at the end, which means by the time we get to the actual "6s at the end" case it may not be testing anything new